File: code/smv-debugger/src/main/java/org/fbme/smvDebugger/panel/condition/ImpossibleComparsionException.java
package org.fbme.smvDebugger.panel.condition;

public class ImpossibleComparsionException extends ConditionParseException {
  public ImpossibleComparsionException(final String argument, final String expression) {
    super(String.format("Argument '%s' can't be compared with complex expression '%s'", argument, expression));
  }
}

File: code/smv-debugger/src/main/kotlin/org/fbme/smvDebugger/panel/condition/ImpossibleComparsionException.kt
package org.fbme.smvDebugger.panel.condition

class ImpossibleComparsionException(argument: String?, expression: String?) : ConditionParseException(
    String.format(
        "Argument '%s' can't be compared with complex expression '%s'",
        argument,
        expression
    )
)
